data/pigui/modules/station-view/01-lobby.lua

```diff
@@ -49,7 +50,15 @@ local hydrogenIcon = PiImage.New("icons/goods/Hydrogen.png")
 local hyperdriveIcon = PiImage.New("icons/goods/Hydrogen.png")
 
 local popupMsg = ''
-local popupId = 'lobbyPopup'
+local popup
+popup = ModalWindow.New('lobbyPopup', function()
```

Comment:
Same issue here, please prefer the `local popup = ModalWindow.New` style over split lines.

Reply:
`local function funcName()` worked for allowing a recursive call, but I'm not sure there's a way around it here. `popup` needs to be at least declared to be referenced inside the render function.